spec/integration/util/windows/security_spec.rb

```diff
@@ -407,14 +418,18 @@ def grant_everyone_full_access(path)
         end
 
         describe "#group=" do
+          it "should accept the test group" do
+            check_group_depending_on_current_user(path)
+          end
+
           it "should accept a group sid" do
             winsec.set_group(sids[:power_users], path)
             expect(winsec.get_group(path)).to eq(sids[:power_users])
           end
 
           it "should accept a user sid" do
-            winsec.set_group(sids[:admin], path)
-            expect(winsec.get_group(path)).to eq(sids[:admin])
+            winsec.set_group(sids[:current_user], path)
+            expect(winsec.get_group(path)).to eq(sids[:current_user])
```

Comment:
Line 383 (`set_group_depending_on_current_user(path)`) will set_group to guest (or a restricted permission set). I added line 422 (`check_group_depending_on_current_user(path)`) to verify it succeeded.